### DISCUSSION TITLE
Reporting results on my virtualization environment

### DISCUSSION THREAD

**Discussion by Dan-Essin** [helloSystem/hello#497] [Q&A]
as requested, my experience today
VirtualBox 7.0.6
VM:  8gb ram, 4 cpu, efi on, usb 3.0, bridged network (nat never works), 35gb disk
I got the language selection screen and clicked english. Then waited
my screen after waiting 20 minutes
![2023-04-01_15h32_57](https://user-images.githubusercontent.com/1776053/229318461-a5b549d2-5181-4696-9ac7-cd25477d836d.png)

It doesn't seem like this works.
Any suggestions?

**Comment by probonopd** [comment]
Which version of the ISO are you using?
It does work on VirtualBox 6 on FreeBSD hosts; I never tested VirtualBox 7 on Windows.

If you got the language selection screen, this means that Xorg was running, then seemingly crashed. Try playing with the graphics settings in VirtualBox, maybe it helps.

In general, it's easy to run helloSystem on real hardware (not in a VM), e.g., by using a Live USB stick or [Ventoy](https://www.ventoy.net/). Maybe this works better for you.

**Comment by mchasard** [comment]
i have the same issue under virtualbox last iso ...which graphics settings should i user ?

**Reply by probonopd** [reply]
I am using VMSVGA, 256 MB graphics memory.

**Reply by mchasard** [reply]
Thanks for this answer i have set VMSVGA and 128MB is the max for me but it seeems to be better ....
